File: giliboim-backend/src/main/java/com/kh/giliboim/account/model/vo/Inquire.java
package com.kh.giliboim.account.model.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Inquire {
	
    private int inquireNo;
    private int memberNo;
    private String title;
    private String inquireContent;
    private String createDate;
    private String status;
    private String answerContent;

}

File: giliboim-frontend/src/type/inquire.ts

// 문의 타입 
export type Inquire = {
    inquireNo :number,
    memberNo: number,
    title : string,
    inquireContent : string, 
    createDate : string,
    status:string, 
    answerContent : string
}

export type Inquires = Inquire[];

// 문의 초기값 

export const initInquire = {
    inquireNo :0,
    memberNo: 0, 
    title : '',
    inquireContent : '', 
    createDate : '',
    status: '',
    answerContent : ''
}


